README.md

```diff
@@ -1844,3 +1777,16 @@ make -C lambdas/service local
 The script serves the Swagger editor locally at a URL where your current version
 of the API documentation is visible. Change the docs in `azul/service/app.py`,
 save, refresh the page, and your changes will appear immediately.
+
+
+## 13.2 Tracking changes to the OpenAPI definition document
```

Comment:
```suggestion
## 13.2 Tracking changes to the OpenAPI definition
```

Reply:
I'll change this, but i think the "official" term is document. 
https://swagger.io/specification/#definitions